### DISCUSSION TITLE
Empêcher les joueurs hors classement d'apparaître dans les events dons d'argent

### DISCUSSION THREAD

**Discussion by Thesmokingllama** [Crownicles/Crownicles#803] [Nouvelles idées]
![87476966-E5FA-4C43-830A-C8459E550744](https://user-images.githubusercontent.com/80409766/147224869-1594e7f5-5d79-49d0-867a-3e5e18e233cb.jpeg)

**Comment by BastLast** [comment]
j'irai meme jusqua dire que ca serait que pour les joueur du top week

**Comment by Procion06** [comment]
Dans la plupart des events non ?
C'est comme se faire bousculer ou croiser un joueur inactif/hors classement sur le chemin pendant un trajet
